bikeshed/InputSource.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
```

Comment:
Improves the syntax for type annotations of methods like `relative()` that return the type of the class they appear inside of. I can drop this and name those types with a string.

https://www.python.org/dev/peps/pep-0563/

Reply:
Looking over the PEP, I'm fine with this.